src/main/java/com/zufar/onlinestore/cart/endpoint/CartEndpoint.java

```diff
@@ -30,14 +31,14 @@
 @RequiredArgsConstructor
 @Validated
 @RequestMapping(value = CartEndpoint.CART_URL)
-public class CartEndpoint {
+public class CartEndpoint implements CartsApi {
 
     public static final String CART_URL = "/api/v1/cart";
 
     private final CartApi cartApi;
 
     @PostMapping(value = "/items")
-    public ResponseEntity<ShoppingSessionDto> addNewItemToShoppingSession(@RequestBody @Valid final AddNewItemsToShoppingSessionRequest request) {
+    public ResponseEntity<ShoppingSessionDto> addNewItemToShoppingSession(@RequestBody final AddNewItemsToShoppingSessionRequest request) {
```

Comment:
Why did you delete ‘@Valid'?
Rollback it, please

Reply:
With this annotation we have a trouble in case when we try to use our API not in IDE (i use Postman): Hibernate validator HV000151 "A method overriding another method must not alter the parameter constraint configuration").
In our generated interface we have the same annotation, its incorrect.
More info: https://forum.hibernate.org/viewtopic.php?p=2469903